Dockerfile

```diff
@@ -0,0 +1,12 @@
+# From https://community.gitpod.io/t/is-there-an-easy-way-to-debug-a-vscode-extension-inside-gitpod/596/63
+FROM gitpod/workspace-full-vnc
```

Comment:
I'm not sure if all of this is necessary since GitPod now supports debugging extensions natively. I'll try to find a simpler base image, maybe just one with Node?